src/core/Song.cpp

```diff
@@ -1208,11 +1208,7 @@ void Song::loadProject( const QString & fileName )
 		}
 		else
 		{
-#if (QT_VERSION >= QT_VERSION_CHECK(5,15,0))
-			QTextStream(stderr) << Engine::getSong()->errorSummary() << Qt::endl;
-#else
```

Comment:
I didn't even realize this was being removed. This was added in the previous deprecation PR. Let's not remove it.

Reply:
My bad, I thought I added this for some reason. Adding the necessary commits soon.